CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * New grayscale style. This also add two new cmaps `cet_grey_r` and `cet_grey_r`. These are perceptually uniform gray scale cmaps from colorcet (linear_grey_10_95_c0) (#1164)
 * Add warmup groups to InferenceData objects, initial support for PyStan (#1126) and PyMC3 (#1171)
 
```

Comment:
Being picky here. Looks like theres a blank line that can be removed

Reply:
Is this changelog item correct? It feels like this PR is internally facing readability changes (which is great) but the feature described here ie something else?

Reply:
It was a duplicated line. Thank you for your helpful comment